memo_front/urls.py

```diff
@@ -1,9 +1,9 @@
 from django.urls import path
-from .views import card_list, assigment
+from .views import deck, assigment
 
 app_name = "memo_front"
 
 urlpatterns = [
-    path("list/<slug:deck_id>/", card_list, name="card_list"),
+    path("list/<slug:deck_id>/", deck, name="deck"),
```

Comment:
I think you should rename this url to `decks/<slug:deck_id>/` for keeping consistency.